index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@0;1&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sign Up</title>
+</head>
+
+<body>
+
+  <div class="singin-button">
+    <div class="signin-info">
+      <p>Already have an Intuit account?</p>
+      <a href="">I forgot my userID or Password</a>
+    </div>
+    <div>
+      <button class="sign-in-button"><img class="signin-logo" src="img/signinlogo.png" alt=""> Sign In</button>
+    </div>
+  </div>
+
+  <main>
+      <div class="main-logo">
+          <img src="img/mainlogo.png" alt="main-logo">
+      </div>
+      <div class="side-logos">
+       <img src="img/mint.png" alt="nmint-logo">
+       <img src="img/quick.png" alt="quick-logo">
+       <img src="img/turbo.png" alt="turbo-logo">
+      </div>
```

Comment:
Please follow the following guidelines
- [ ] It shows the Mint logo and text (logo on the left, text on the right)

- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right)

- [ ]  It shows the Turbotax logo and text (logo on the left, text on the right)

Reply:
On the original page, they used SVG files, not text, so we could not find the original font so we used the original SVG file that we found on mint's login page.